application/src/test/java/org/opentripplanner/apis/gtfs/GraphQLIntegrationTest.java

```diff
@@ -226,6 +269,29 @@ public Set<Route> getRoutesForStop(StopLocation stop) {
     };
     routes.forEach(transitService::addRoutes);
 
+    timetableRepository.setTransitLayer(
```

Comment:
Rather than doing this complicated setup in the GTFS test you should write them as module tests, like these: https://github.com/opentripplanner/OpenTripPlanner/blob/a8a1730f73df91794295f1db89e5dfd27735a730/application/src/test/java/org/opentripplanner/updater/trip/moduletests/delay/DelayedTest.java#L26-L68

There you can write the test on the service rather than the API.

You can then mock the returned data in the API test (or not test it at all).

Reply:
Can you please help me to get the PR on the GTFS-RT spec approved so that I don't risk messing up with my other work?

Reply:
I think it is better for me to defer the replacement module test in #6028 as I will add the ability to specified skipped stops.